spec/acceptance/user_manages_functions_spec.rb

```diff
@@ -34,4 +34,36 @@
     result = execute("SELECT * FROM test() AS result")
     expect(result).to eq("result" => "testest")
   end
+
+  it "handles functions with arguments" do
+    successfully "rails generate fx:function adder"
+    write_function_definition "adder_v01", <<-EOS
+      CREATE FUNCTION adder(x int, y int)
+      RETURNS int AS $$
+      BEGIN
+          RETURN $1 + $2;
+      END;
+      $$ LANGUAGE plpgsql;
+    EOS
+    successfully "rake db:migrate"
+
+    result = execute("SELECT * FROM adder(1, 2) AS result")
+    # require "byebug"
```

Comment:
Could you remove these debug statements and reformat the test file to follow how the test suite is formatted?

Reply:
I removed the debug statments. Not sure what you ment with reformating. The spec is copied from https://github.com/teoljungberg/fx/pull/8/files